### ISSUE TITLE
mod outdated

### ISSUE THREAD

**Issue by Pandurr** [skairunner/sky-oni-mods#57]
please fix <3

**Comment by digital-pet** [comment]
The great merge appears to have rendered most mods outdated as the base game codebase has been replaced with the DLC and the modding API has been updated. Looks like it's time for that DLC compatibility sweep.

**Comment by vizthex123** [comment]
According to the [Steam page](https://steamcommunity.com/sharedfiles/filedetails/?id=1873476551), these mods won't be updated until the DLC is fully released.

**Comment by digital-pet** [comment]
Right, but that was when the DLC was using a different codebase entirely and it didn't make sense to adapt. Now that the base game has been made incompatible with this mod because everything is using the DLC codebase, it doesn't make sense not to update it for Harmony 2.0
